troubleshooting/troubleshooting-cannot-receive-push-notifications-on-android-when-the-app-is-closed.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Cannot Receive Push Notifications on Android When the App Is Closed
+description: On Android, push notifications are not received when the app is not running (closed). Notifications are received when the app is running either in the foreground or in the background.  
+type: troubleshooting
+page_title: Push Notifications Are Not Received on Android When the App Is Closed 
+slug: cannot-receive-push-notifications-on-android-when-the-app-is-closed
+position:
+tags: push, Android, push notifications, GCM, Xiaomi, Huawei, MIUI
```

Comment:
New tags should be added to the tags.md document as well. This will allow tag filtering in the future.